pages/common/doctl-auth.md

```diff
@@ -0,0 +1,24 @@
+# doctl auth
+
+> Authenticate doctl with one or more API tokens.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/auth/>.
+
+- Show available commands:
+
+`doctl auth --help`
```

Comment:
Examples in tldr pages should be ordered in the order people might use them. Let's put this example at the bottom of the page.